docs/_manage/integration-changes/rollout-cadence-iq-health-tips.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Rollout cadence and integration quality health tips
+order: 15
+layout: post-toc
+---
+
+# Rollout Cadence and integration quality health tips
+
+Our shared customers rely on Zapier and your integration for business-critical workflows. Addressing feedback early and often ensures users have the best experience, both with Zapier's platform and yours. Follow these tips on how.
+
+## 1. Embrace early bug resolution
+Early bug resolution prevents minor issues from escalating into significant problems, ensuring your users have a smooth and reliable experience.
+
+## 2. Prioritize user-requested features
+Requests for new features are invaluable insights into your user needs and pain points. Prioritizing these requests in your development roadmap can help enhance the functionality of your integration and significantly boost user retention and loyalty.
+
+> Pro tip: Explore users' most valued and widely used features per app category! Head over to our [must-have triggers and actions](https://platform.zapier.com/build/recommended-integration-features) page for details.
+
+## 3. Make Zapier an integral part of your product strategy
+Your integration is a powerful tool for automating workflow, increasing productivity, and providing value to your users. Including it in your product expansion strategy and product roadmap planning, ensures it evolves in tandem with your core offerings.
+
+> Pro tip: Read on how you can [prioritize user feedback](https://zapier.com/blog/prioritize-user-feedback-with-beamer/) in your product roadmap planning.
+
+## 4. Spread the word about your integration's new features
+Highlight new features and enhancements to your integration in your marketing materials. Adding these to newsletters, blogs, and in-product messages (IPMs), can help drive engagement and encourage users to explore new functionalities, leading to increased adoption and deeper integration into their daily workflows.
+
+> Pro Tip: Get your integration in front of more users by embedding Zapier’s [Partner Solutions](https://zapier.com/partner/solutions/plug-and-play) directly into your product. 
+
+## 5. Keep Your Integration Fresh with Regular Updates
```

Comment:
Keep your integration fresh with regular updates
(sentence case to follow developer style guide from Tech Writing)